packages/widgets/src/components/CollapsibleNavBar/CollapsibleNavBar.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import { fireEvent } from '@testing-library/dom';
+import { ITEMS } from './stories/consts';
+import { StoryComponent } from './stories/StoryComponent';
+
+describe('CollapsibleNavBar', () => {
+  it('Should be correctly rendered', () => {
+    const { queryByText, getAllByText } = render(
+      <StoryComponent items={ITEMS} />,
+    );
+
+    queryByText('Dashboard');
+    queryByText('Bots');
+    queryByText('Statistics');
+    queryByText('Max in Work');
+    queryByText('History');
+    getAllByText('Settings');
+  });
+
+  it('Should be expanded the group', () => {
+    const { container } = render(<StoryComponent items={ITEMS} />);
+
+    const statisticsArrow = container.querySelector(
+      'button#chartstatisticsaccordion',
+    );
+
+    expect(statisticsArrow as Node).toHaveAttribute('aria-expanded', 'false');
+
+    fireEvent.click(statisticsArrow as Node);
+
+    expect(statisticsArrow as Node).toHaveAttribute('aria-expanded', 'true');
+  });
+
+  it('Sidebar should be expanded', () => {
```

Comment:
It reads like this: `it sidebar should be expanded`. Please correct. Maybe something like this should work:
`It should have the Sidebar expanded`

Reply:
Or even `it should be expanded`